### PR TITLE
ci: Update .lock files in dependabot PRs

### PR DESCRIPTION
**Pull request by witwash**
#### Summary

Added a follow-up action to commit `.lock` files if PR was created by the `dependabot`.

#### Testing steps

`Dependabot` PR should trigger this action and changes in `.lock` should be committed.

#### Follow-up issues

None

#### Check during review

- Verify against Jira issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).

### CODE REVIEW

**Comment by witwash** [comment]
@ookami-kb,
looks like dependabot is updated and this one is not needed anymore

**Comment by witwash** [comment]
@ookami-kb, what do you think?

File: .github/workflows/dependabot-follow-up.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Update .lock files on dependabot PR
+on:
+  pull_request:
+    types:
+      - opened
```
Comment (ookami-kb) [inline .github/workflows/dependabot-follow-up.yml:5]: PR can also be updated/rebased, I think, it should run in this case as well.

**Review by ookami-kb — COMMENTED** [review]
In general, yeah, makes sense, let's try it this way.

**Comment by witwash** [comment]
If this [one](https://frontside.com/blog/2020-05-26-github-actions-pull_request/) is true,
> When a Github Actions workflow is configured to run on pull requests, by default it will be triggered on three different types of activities: `opened`, `synchronize`, and `reopened`.

 then by default, this action would be triggered on `open`, `reopen`, and `synchronize`. But I could not find it in the docs and decided to define it explicitly.

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**